tests/test_config.py

```diff
@@ -1,174 +1,152 @@
 import pytest
 from pathlib import Path
-from datetime import datetime, timezone
 import re
 
 import jirahub
-from jirahub.config import load_config, generate_config_template, SyncFeature
+from jirahub.config import load_config, generate_config_template, SyncFeature, JirahubConfig, validate_config
 from jirahub.entities import Source
 
 
 CONFIG_PATH = Path(__file__).parent / "configs"
 
 
 def test_load_config_minimal():
-    config = load_config(CONFIG_PATH / "minimal_config.ini")
+    config = load_config(CONFIG_PATH / "minimal_config.py")
 
     assert config.jira.server == "https://test.jira.server"
     assert config.jira.project_key == "TEST"
-    assert config.jira.github_repository_field is None
-    assert config.jira.github_issue_id_field is None
-    assert config.jira.closed_statuses == {"closed"}
+    assert config.jira.github_issue_url_field == "github_issue_url"
+    assert config.jira.jirahub_metadata_field == "jirahub_metadata"
+    assert config.jira.closed_statuses == ["closed"]
     assert config.jira.close_status == "Closed"
     assert config.jira.reopen_status == "Reopened"
     assert config.jira.open_status is None
-
-    assert config.jira.sync.create_issues is False
-    assert config.jira.sync.sync_comments is False
-    assert config.jira.sync.sync_status is False
-    assert config.jira.sync.sync_labels is False
-    assert config.jira.sync.sync_milestones is False
-    assert config.jira.sync.labels == set()
-    assert config.jira.sync.redact_regexes == []
-
-    assert config.jira.filter.min_created_at is None
-    assert config.jira.filter.include_labels == set()
-    assert config.jira.filter.exclude_labels == set()
-    assert config.jira.filter.open_only is True
-
-    assert config.jira.defaults.issue_type == "Story"
-    assert config.jira.defaults.priority is None
-    assert config.jira.defaults.components == set()
+    assert config.jira.sync_comments is False
```

Comment:
Yup, though it can be configured to sync in only one direction.  It's also possible to have multiple github repos sync to the same jira project, but currently that requires one config per github/jira project pair.